api_app/analyzers_manager/observable_analyzers/dns0/dns0_rrsets.py

```diff
@@ -0,0 +1,199 @@
+from logging import getLogger
+from typing import Dict
+
+import requests
+
+from api_app.analyzers_manager import classes
+from api_app.analyzers_manager.exceptions import (
+    AnalyzerConfigurationException,
+    AnalyzerRunException,
+)
+from api_app.analyzers_manager.observable_analyzers.dns0.dns0_base import DNS0Mixin
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = getLogger(__name__)
+
+_supported_dns_types = [
+    "A",
+    1,
+    "AAAA",
+    28,
+    "AFSDB",
+    18,
+    "APL",
+    42,
+    "AXFR",
+    252,
+    "CAA",
+    257,
+    "CDNSKEY",
+    60,
+    "CDS",
+    59,
+    "CERT",
+    37,
+    "CNAME",
+    5,
+    "CSYNC",
+    62,
+    "DHCID",
+    49,
+    "DLV",
+    32769,
+    "DNAME",
+    39,
+    "DNSKEY",
+    48,
+    "DS",
+    43,
+    "EUI48",
+    108,
+    "EUI64",
+    109,
+    "HINFO",
+    13,
+    "HIP",
+    55,
+    "HTTPS",
+    65,
+    "IPSECKEY",
+    45,
+    "IXFR",
+    251,
+    "KEY",
+    25,
+    "KX",
+    36,
+    "LOC",
+    29,
+    "MX",
+    15,
+    "NAPTR",
+    35,
+    "NS",
+    2,
+    "NSEC",
+    47,
+    "NSEC3",
+    50,
+    "NSEC3PARAM",
+    51,
+    "OPENPGPKEY",
+    61,
+    "OPT",
+    41,
+    "PTR",
+    12,
+    "RRSIG",
+    46,
+    "SIG",
+    24,
+    "SMIMEA",
+    53,
+    "SOA",
+    6,
+    "SRV",
+    33,
+    "SSHFP",
+    44,
+    "SVCB",
+    64,
+    "TA",
+    32768,
+    "TKEY",
+    249,
+    "TLSA",
+    52,
+    "TSIG",
+    250,
+    "TXT",
+    16,
+    "URI",
+    256,
+    "ZONEMD",
+    63,
+]
+
+_supported_format_types = [
+    "json",
+    "cof",
+    "dig",
+]
+
+_supported_directions = [
+    "right",
+    "left",
+]
+
+
+class DNS0Rrsets(classes.ObservableAnalyzer, DNS0Mixin):
+    endpoint: str = "rrsets"
```

Comment:
this should not be a parameter, the user should not be able to change it

Reply:
you mean via source code or via intelowl plugin config page ?
because this is not mapped in the `AnalyzerConfig` of this analyzer, it is only used locally in the code to determine the endpoint.

Reply:
you are right, my bad, I got confused after I did read the changed doc
`  - This parameter should not be changed from default value.` but it refers to the `direction` parameter. ok